src/test/java/tictactoe/PlayerFactoryTest.java

```diff
@@ -36,4 +37,37 @@ public void itCreateAComputerPlayerAndComputerPlayer() {
         assertThat(players[1]).isInstanceOf(ComputerPlayer.class);
         assertThat(players[1].getMark()).isEqualTo(MarksEnum.ROUND);
     }
+
+    @Test
+    public void itCreateARandomPlayerAndComputerPlayer() {
+        PlayerFactory playerFactory = new PlayerFactory(null, null);
+        Player[] players = playerFactory.getPlayers(GameTypes.RANDOM_VS_COMPUTER);
+
+        assertThat(players[0]).isInstanceOf(RandomPlayer.class);
+        assertThat(players[0].getMark()).isEqualTo(MarksEnum.CROSS);
+        assertThat(players[1]).isInstanceOf(ComputerPlayer.class);
+        assertThat(players[1].getMark()).isEqualTo(MarksEnum.ROUND);
+    }
+
+    @Test
+    public void itCreateARandomPlayerAndRandomPlayer() {
+        PlayerFactory playerFactory = new PlayerFactory(null, null);
+        Player[] players = playerFactory.getPlayers(GameTypes.RANDOM_VS_RANDOM);
+
+        assertThat(players[0]).isInstanceOf(RandomPlayer.class);
+        assertThat(players[0].getMark()).isEqualTo(MarksEnum.CROSS);
+        assertThat(players[1]).isInstanceOf(RandomPlayer.class);
+        assertThat(players[1].getMark()).isEqualTo(MarksEnum.ROUND);
+    }
+
+    @Test
+    public void itCreateARandomPlayerAndHumanPlayer() {
```

Comment:
It's more grammatically correct to say `itCreate**s**ARando....` (ie an 's' on the end of Create)